src/nationalarchives/components/nested-navigation/nested-navigation.yaml

```diff
@@ -0,0 +1,34 @@
+params:
+  - name: items
+    type: object
+    required: true
+    description: The inputs to describe the file structure
+
+examples:
+  - name: default
+    description: The default nested navigation layout
+    data:
+      items:
+        - name: folder1
+          type: node
+          id: folderid1
+          children:
+            - name: file1
+              type: item
+              id: fileid1
+            - name: file2
+              type: item
+              id: fileid2
+        - name: folder12
+          type: node
+          id: folderid12
+          children:
+            - name: file12
+              type: item
+              id: fileid12
+            - name: file22
+              type: item
+              id: fileid22
+        - name: file1
```

Comment:
Is having the same file name twice in different folders deliberate?

Reply:
No, just lazy typing, I've changed it to something different and updated the tests.